pkg/sysinfo/sysinfo_linux_test.go

```diff
@@ -102,3 +102,13 @@ func TestNumCPU(t *testing.T) {
 		t.Fatal("CPU returned must be greater than zero")
 	}
 }
+
+func TestNumMems(t *testing.T) {
+	if _, err := os.Stat("/proc/*/numa_maps"); !os.IsNotExist(err) {
```

Comment:
```suggestion
	if _, err := os.Stat(**"/proc/*/numa_maps"**); !os.IsNotExist(err) {
```
Does this ever return a non-error? Who is resolving the wild card?

Reply:
Should check /proc/1/numa_maps or /proc/self/numa_maps?

Reply:
@rhatdan @jwhonce I was unsure of which folder to check since multiple contained `numa_maps` /proc/1 definitely contained it. Should I switch to that? Right now all of the tests pass.

Reply:
This test will never run, since the file /proc/*/numa_maps does not exist.  You probably see `NUMA must be supported` messages in logs.

Better to just do /proc/self/numa_maps ...